doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md

```diff
@@ -45,58 +45,53 @@ This is done at the cygnus-ngsi Http listeners (in Flume jergon, sources) thanks
 
 [Top](#top)
 
-### <a name="section1.2"></a>Mapping `NGSIEvent`s to PostgreSQL data structures
-PostgreSQL organizes the data in schemas inside a database that contain tables of data rows. Such organization is exploited by `NGSIPostgreSQLSink` each time a `NGSIEvent` is going to be persisted.
+### <a name="section1.2">PostgreSQL data models</a>
+PostgreSQL organizes the data in schemas inside a database that contain tables of data rows. Such organization is exploited by `NGSIPostgreSQLSink` each time a `NGSIEvent` is going to be persisted, by mapping `NGSIEvent`s to PostgreSQL data structures. 
+
+The name of these tables, schemas and databases depends on the configured data model, selected by defining the parameter `cygnus-ngsi.sinks.postgresql-sink.data_model` in the [`agent.conf`](https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-ngsi/installation_and_administration_guide/ngsi_agent_conf.md) file or by defining the env var `CYGNUS_POSTGRESQL_DATA_MODEL`.
+
+For a given entity, `entityname`, of type `entitytype`, inside a tenant `service` under `/servicepath`, how the information is stored in the database differs from using different datamodel configurations. The following table contains the database, schema and table names used for each datamodel supported by this sink:
+
+| datamodel name                            | database name | schema  name | table name                        | note                                              |
+|-------------------------------------------|---------------|--------------|-----------------------------------|---------------------------------------------------|
+| by default                                | `agent.conf`  | service      | servicepath_entityname_entitytype | Automatic schema creation                         |
+| `dm-by-fixed-entity-type-database-schema` | service       | _servicepath | entitytype                        | Database and schema need to be created in advance |
```

Comment:
The initial underscore `_` in `-schema` DM is a bit weird (although consistent with the code implementation)

Checking with @manucarrace before merging this PR.

Reply:
Discussed with @manucarrace by chat

He was unaware of this. The `-schema` DMs are not used.

An issue has been created to analyze this: https://github.com/telefonicaid/fiware-cygnus/issues/2201